tm2/pkg/std/coin.go

```diff
@@ -616,8 +616,7 @@ func (coins Coins) Sort() Coins {
 // Parsing
 
 var (
-	// Denominations can be 3 ~ 16 characters long.
-	reDnmString = `[a-z][a-z0-9]{2,15}`
+	reDnmString = `[a-z\/][a-z0-9_.:\/]{2,}` // TODO: limit pkg_path length // REF: https://github.com/gnolang/gno/pull/875#discussion_r1543384275
```

Comment:
I've created an issue, let's remove the todo

https://github.com/gnolang/gno/issues/2053

Reply:
362cd1cc6a33dc63412c30149422d5e03c212f14